packages/common-all/src/analytics.ts

```diff
@@ -32,6 +32,7 @@ export enum CLIEvents {
 
 export enum TutorialEvents {
   WelcomeShow = "WelcomeShow",
+  ClickStart = "ClickStart",
```

Comment:
nit: how about we start using past tense on the events?  Maybe "Getting_Started_Clicked" - may also be good to be slightly more specific here, in case in the future we have various 'start' buttons.